queries/gdscript/highlights.scm

```diff
@@ -0,0 +1,27 @@
+(identifier) @variable
+
+(name) @variable
+(type) @type
+(comment) @comment
+
+(enum_definition
+  (name) @constant) @Statement
```

Comment:
We don't have `@Statement`

Reply:
We actually do, it is a shortcut to set highlights directly from their groups, not recommended though...

Reply:
I was not trying to say that it won't work. It's just not in our `TSHighlights`. If there is a strong reason to use it that is very language specific I'd say it ok.

Reply:
I think this would set `TSConstant` as well as the `Statement` groups.